sniff.js

```diff
@@ -65,6 +65,11 @@ define(["./features"], function (has) {
 		has.add("mac", dav.indexOf("Macintosh") >= 0);
 
 		has.add("msapp", parseFloat(dua.split("MSAppHost/")[1]) || undefined);
+
+		var wp = parseFloat(dua.split("Windows Phone ")[1]) || undefined;
+		if (wp) {
+			has.add("wp", wp);
+		}
```

Comment:
Ultimately we should be consistent and share the same approach across the has values (just adobe mac & msapp are using a different strategy)